jscomp/bsb/bsb_clean.ml

```diff
@@ -39,7 +39,8 @@ let ninja_clean  proj_dir =
     Bsb_log.warn "@{<warning>ninja clean failed@} : %s @." (Printexc.to_string e)
 
 let clean_bs_garbage proj_dir =
-  Bsb_log.info "@{<info>Cleaning:@} in %s@." proj_dir ;
+  let proj_dir = Bsb_build_util.get_build_artifacts_location proj_dir in
```

Comment:
Since you call this function below, inside `clean_bs_deps`, you might not need to call it here.